docs/legacy/common-problems.asciidoc

```diff
@@ -253,11 +250,14 @@ APM agent --> Load Balancer  --> APM Server
 
 *If Elasticsearch is down*
 
-If Elasticsearch goes down, the APM Server will keep data in memory until Elasticsearch is back up,
-or until it runs out of space in its internal in-memory queue.
-You can <<adjust-queue-size,adjust the internal queue size>> if necessary.
-When the queue becomes full, APM Server will respond with <<queue-full,`HTTP 503: Queue is full`>>,
-and data will be lost.
+// Need to update this with dev feedback
+// If ES goes down, is all data lost?
+
+// If Elasticsearch goes down, the APM Server will keep data in memory until Elasticsearch is back up,
+// or until it runs out of space in its internal in-memory queue.
+// You can <<adjust-queue-size,adjust the internal queue size>> if necessary.
+// When the queue becomes full, APM Server will respond with <<queue-full,`HTTP 503: Queue is full`>>,
+// and data will be lost.
```

Comment:
Since the queue is gone in 8.0+, does that mean that if ES goes down, all data is lost?

Reply:
The queue isn't gone, it's just different :)

There's still an in-memory queue. If it fills up, the server will block until there is space, or the request timeout is reached.

The new queue shouldn't need tuning. I'd suggest dropping the "adjust the internal queue size" link and section.

Reply:
Ahhh. Thanks! Fixed in 780f7a3.